eZ/Bundle/EzPublishIOBundle/DependencyInjection/ConfigurationFactory/Flysystem.php

```diff
@@ -62,7 +62,8 @@ public function configureHandler(ServiceDefinition $definition, array $config)
     private function createFilesystem(ContainerBuilder $container, $name, $adapter)
     {
         $adapterId = sprintf('oneup_flysystem.%s_adapter', $adapter);
-        if (!$container->hasDefinition($adapterId)) {
+        // has either definition or alias
+        if (!$container->has($adapterId)) {
```

Comment:
Custom adapter is exposed by Oneup as an alias, hence this adjustment is required.